apisix/plugins/consumer-restriction.lua

```diff
@@ -95,6 +99,10 @@ local function is_include(value, tab)
 end
 
 local function is_method_allowed(allowed_methods, method, user)
+    if not user then
+        return false
+    end
```

Comment:
I don't think the logic here is correct, if the blacklist exists and we can't get the user info, the request should be forbidden